### DISCUSSION TITLE
A sharp, high grade preset demo! and a new preset!

### DISCUSSION THREAD

**Discussion by Degamisu** [ntsc-rs/ntsc-rs#167] [Presets]
A lot of my presets are a blurry, fuzzy mess. In reality if you have a good VCR deck, it wouldn't be _really_ blurry. The tape preset is still in development, but the old tape (1998 T-120 tape) will be available to download. The old tape is still *highly* realistic, its just a different technique than what I'm experimenting with right now.

## 1998 T-120 preset (Old technique)
>[!NOTE]
>Notice the blurriness and overall smearing. It is extremely apparent.

![image](https://github.com/user-attachments/assets/1dfe234d-5e76-4960-b888-c400bf69ec47)

## Fuji EHG T120 (New technique)
>[!NOTE]
>Notice how sharp and clear this is. It is based off of a Fuji EHG tape, from a reference video [here](https://www.youtube.com/watch?v=pWjsOr0Q87s)

![image](https://github.com/user-attachments/assets/1abeb877-4105-4791-85ac-bc1d650b509c)

https://github.com/user-attachments/assets/00812db7-c2c3-4ec6-bc47-06b5fcc9c05a

### Update: Added a bad tracking version.

![image](https://github.com/user-attachments/assets/8453e1f4-57a2-4fc3-801e-40701e70f001)

These presets are subject to change in the future.

Indev version of 1998 T120: [1998-T120.json](https://github.com/user-attachments/files/16736977/1998-T120.json)

**Comment by aether1941** [comment]
still waiting for these honestly.
